fem/restriction.hpp

```diff
@@ -171,6 +177,14 @@ class FaceRestriction : public Operator
    */
    void Mult(const Vector &x, Vector &y) const override = 0;
 
+   virtual void MultUnsigned(const Vector &x, Vector &y) const
+   {
+      MFEM_ABORT("MultUnsigned not implemented yet!");
```

Comment:
Not quite accurate since derived class `ConformingFaceRestriction` overrides this.

If `AbsMult` is going to be the standard terminology, why not define `MultUnsigned` to call `AbsMult` (for which the default base class implementation will simply error), and then you can override `ConformingFaceRestriction::AbsMult`?